packages/web-pixels-extension/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@shopify/web-pixels-extension",
   "description": "Provides tools to author Web Pixels extension",
-  "version": "0.4.22",
+  "version": "0.5.0",
```

Comment:
Looks good! Seems like a true first major update in a while.

BTW, so we're good with [exposing the meta type](https://github.com/Shopify/ui-extensions/pull/1691/files#diff-fddd73774ed30e13140b787531539f00abec07bf666e9b89d08a1a135d8c8fc1R457) right?

Reply:
Also, I thought we were going to [remove this line from the PR](https://github.com/Shopify/ui-extensions/pull/1691/files#diff-250cae8251833df357522e5b9350bf587e1f1833f971fd51ab0e66df0e6b3e6aR7)? 

Maybe Github is bugging out, but it seems like it was merged?

Reply:
We're ok with shipping this as is for now, the types in question are not used. We have an issue to omit certain types in the future.